install/kubernetes/helm/istio/charts/mixer/templates/config.yaml

```diff
@@ -216,16 +219,49 @@ spec:
     responseCode: response.code | 0
     responseSize: response.size | 0
     requestSize: request.size | 0
+    requestId: request.headers["x-request-id"] | ""
+    clientTraceId: request.headers["x-client-trace-id"] | ""
     latency: response.duration | "0ms"
     connectionMtls: connection.mtls | false
     userAgent: request.useragent | ""
-    responseTimestamp: response.time
+    responseTimestamp: response.time | timestamp("2017-01-01T00:00:00Z")
     receivedBytes: request.total_size | connection.received.bytes | 0
     sentBytes: response.total_size | connection.sent.bytes | 0
     referer: request.referer | ""
   monitored_resource_type: '"global"'
 ---
 apiVersion: "config.istio.io/v1alpha2"
+kind: logentry
+metadata:
+  name: tcpaccesslog
+  namespace: {{ .Release.Namespace }}
+spec:
+  severity: '"Info"'
+  timestamp: context.time | context.timestamp | timestamp("2017-01-01T00:00:00Z")
+  variables:
+    sourceIp: source.ip | ip("0.0.0.0")
+    sourceApp: source.labels["app"] | ""
+    sourcePrincipal: source.principal | ""
+    sourceName: source.name | ""
+    sourceWorkload: source.workload.name | ""
+    sourceNamespace: source.namespace | ""
+    sourceOwner: source.owner | ""
+    destinationApp: destination.labels["app"] | ""
+    destinationIp: destination.ip | ip("0.0.0.0")
+    destinationServiceHost: destination.service.host | ""
+    destinationWorkload: destination.workload.name | ""
+    destinationName: destination.name | ""
+    destinationNamespace: destination.namespace | ""
+    destinationOwner: destination.owner | ""
+    destinationPrincipal: destination.principal | ""
+    protocol: context.protocol | "tcp"
+    latency: connection.duration | "0ms"
+    connectionMtls: connection.mtls | false
+    receivedBytes: connection.received.bytes | 0
```

Comment:
i think for closed events, we want `connection.received.total_bytes` and `connection.sent.total_bytes`, right?

Reply:
Thanks for catching and for open events these will b 0